packages/block-library/src/navigation/index.php

```diff
@@ -537,7 +537,7 @@ private static function get_responsive_container_markup( $attributes, $inner_blo
 			$toggle_aria_label_close,
 			esc_attr( implode( ' ', $responsive_container_classes ) ),
 			esc_attr( implode( ' ', $open_button_classes ) ),
-			esc_attr( safecss_filter_attr( $colors['overlay_inline_styles'] ) ),
+			( ! empty( esc_attr( safecss_filter_attr( $colors['overlay_inline_styles'] ) ) ) ) ? ( 'style="' . esc_attr( safecss_filter_attr( $colors['overlay_inline_styles'] ) ) . '"' ) : '',
```

Comment:
This works, but a couple of suggestions to make the code more readable. 

- The styles could be assigned to a variable first to avoid escaping/filtering the value twice.
- In PHP, if you use double quotes, you can expand variables within the string

```php
$overlay_inline_styles = esc_attr( safecss_filter_attr( $colors['overlay_inline_styles'] ) );

// ...

! empty( $overlay_inline_styles  ) ? "style=$overlay_inline_styles" : '',
```

Reply:
@talldan updated.